src/Symfony/Component/Serializer/Tests/Normalizer/ObjectNormalizerTest.php

```diff
@@ -892,6 +892,23 @@ public function testDenormalizeWithIgnoreAttributeAndPrivateProperties()
 
         $this->assertEquals($expected, $obj);
     }
+
+    public function testNormalizeWithMethodNamesSimilarToAccessors()
+    {
+        $classMetadataFactory = new ClassMetadataFactory(new AttributeLoader());
+        $normalizer = new ObjectNormalizer($classMetadataFactory);
+
+        $object = new ObjectWithAccessorishMethods();
+        $normalized = $normalizer->normalize($object);
+
+        $this->assertFalse($object->isAccessorishCalled());
```

Comment:
We have two possibilities to set `$accesorishCalled` property to true, we don't know which one exactly, right?

Wouldn't it be better to use this property as a counter and assert a count of 2? This way we know, `cancel()` and `hash()` would be called? Or even more precise would be to have a property for `cancel()` and one for `hash()`

Reply:
Ah nevermind, we assert it is still false and so never called.. all good